src/cascadia/TerminalApp/MinMaxCloseControl.h

```diff
@@ -13,16 +13,19 @@ namespace winrt::TerminalApp::implementation
 {
     struct MinMaxCloseControl : MinMaxCloseControlT<MinMaxCloseControl>
     {
-        MinMaxCloseControl(uint64_t hWnd);
+        MinMaxCloseControl();
 
         void Minimize_Click(winrt::Windows::Foundation::IInspectable const& sender, winrt::Windows::UI::Xaml::RoutedEventArgs const& e);
         void Maximize_Click(winrt::Windows::Foundation::IInspectable const& sender, winrt::Windows::UI::Xaml::RoutedEventArgs const& e);
         void Close_Click(winrt::Windows::Foundation::IInspectable const& sender, winrt::Windows::UI::Xaml::RoutedEventArgs const& e);
         void DragBar_DoubleTapped(winrt::Windows::Foundation::IInspectable const& sender, winrt::Windows::UI::Xaml::Input::DoubleTappedRoutedEventArgs const& e);
 
+        uint64_t ParentWindowHandle() const;
+        void ParentWindowHandle(uint64_t handle);
+
     private:
         void _OnMaximize(byte flag);
-        HWND _window = nullptr;
+        wil::unique_hwnd _window;

```

Comment:
Woah hold up are we sure that we want to be transfering ownership of the HWND to the MinMaxControl? 

What happens when a `unique_hwnd` is released?

Reply:
whoa whoa whoa that was dumb of me
thanks!